code/game/objects/items/robot/robot_items.dm

```diff
@@ -191,7 +191,7 @@ obj/item/borg/stun/attack(mob/M as mob, mob/living/silicon/robot/user as mob)
 			M.dizziness += 5
 			M.confused +=  5
 			M.Jitter(5)
-			add_gamelogs(user, "alarmed [key_name(M)] with \the [src]", tp_link = FALSE)
+			add_gamelogs(user, "alarmed [key_name(M)] with \the [src]", admin = FALSE, tp_link = FALSE)
 		cooldown = world.time + 20 SECONDS
 		add_gamelogs(user, "used \the [src]", admin = TRUE, tp_link = TRUE, tp_link_short = FALSE, span_class = "notice")
```

Comment:
move this to between 183 and 184 and I'll merge this, it makes more sense for the logs to list this before the specific people alarmed